sklearn/preprocessing/tests/test_label.py

```diff
@@ -171,13 +171,14 @@ def test_label_binarizer_errors():
 
     # Fail on y_type
     err_msg = "foo format is not supported"
-    with pytest.raises(ValueError, match=err_msg):
-        _inverse_binarize_thresholding(
-            y=csr_matrix([[1, 2], [2, 1]]),
-            output_type="foo",
-            classes=[1, 2],
-            threshold=0,
-        )
+    for csr_container in CSR_CONTAINERS:
```

Comment:
How about splitting the two loops that are using spare containers into a separate test and then parametrizing that? I think these tests are not dependent on anything else within this test.

Reply:
Can't this loop also be moved into the added test `test_label_binarizer_sparse_errors`?

Reply:
Yes, forgot about it